components/types.ts

```diff
@@ -3,6 +3,11 @@ import m from "mithril";
 
 export const THEME_DEFAULT = "the-default";
 
+export const DefaultFormOptions = {
```

Comment:
This looks wrong

- constants, classes and types start with an uppercase letter. All others should start lowercased
- defaultOptions should not be within types, where only types reside (THEME_DEFAULT should probably move, too)

Reply:
Why did you remove the two defaultOptions from the components? Duplication here is totally ok, since they may change individually in the future.

Reply:
I thought it would stay the same and wanted to remove duplicates - I'll change it back, as you said it can change in the future